src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -202,26 +202,24 @@ impl Command {
         // emscripten has no signal support.
         #[cfg(not(any(target_os = "emscripten")))]
         {
-            use crate::mem;
+            use crate::mem::MaybeUninit;
             // Reset signal handling so the child process starts in a
             // standardized state. libstd ignores SIGPIPE, and signal-handling
             // libraries often set a mask. Child processes inherit ignored
             // signals and the signal mask from their parent, but most
             // UNIX programs do not reset these things on their own, so we
             // need to clean things up now to avoid confusing the program
             // we're about to run.
-            let mut set: libc::sigset_t = mem::uninitialized();
+            let mut set = MaybeUninit::<libc::sigset_t>::uninit();
             if cfg!(target_os = "android") {
                 // Implementing sigemptyset allow us to support older Android
                 // versions. See the comment about Android and sig* functions in
                 // process_common.rs
-                libc::memset(&mut set as *mut _ as *mut _,
-                             0,
-                             mem::size_of::<libc::sigset_t>());
+                set.as_mut_ptr().write_bytes(0u8, 1);
```

Comment:
Can't this call the `sigemptyset` that you also changed? Seems nicer than duplicating the code.

Reply:
The comment above it references this explanation:
https://github.com/rust-lang/rust/blob/d3e2cec29225a46298ec4ebf082f34ebd7cfeecf/src/libstd/sys/unix/process/process_common.rs#L424-L427

Reply:
Oh, nevermind I see what you mean, just call across the two modules. I suppose that's possible, but the one in `process_common.rs` is under `#[cfg(test)]`. It's possible to shuffle that around, but IMO not really worth it for a one-liner.

Reply:
Sure, but as that comment says we always define our own `sigemptyset` on `target_os = "android"`. So why can't we use *that*?

Reply:
Oh wait, I didn't realize that was inside a `mod tests`. Never mind then.